dbt_platform_helper/commands/config.py

```diff
@@ -98,7 +99,9 @@ def deployment():
 
     recommendations = {}
 
-    config_file_check()
+    config_provider = ConfigProvider(ConfigValidator())
+
+    config_provider.config_file_check()
```

Comment:
Seeing this makes me think - maybe we should allow creation of a ConfigProvider without ConfigValidator and that's like saying, don't bother validating.  In this case, there is no validation since it is only calling the config_file_check function